mage_ai/data_integrations/utils/config.py

```diff
@@ -1,5 +1,6 @@
 from jinja2 import Template
 from mage_ai.data_integrations.utils.parsers import NoDatesSafeLoader
+from mage_ai.services.aws.secrets_manager.secrets_manager import get_secret
```

Comment:
in the future, we'll add support to get secrets from gcp secret manager as well. how can we differentiate secrets from aws and gcp?

Reply:
Just saw your description in PR. Yes, let's do `aws_secrets_var`.

Reply:
@wangxiaoyou1993 updated

Reply:
If we import `aws.secrets_manager.secrets_manager` here, `boto3` and `botocore` are required packages. How can we make those packages optional?

Reply:
@wangxiaoyou1993 I moved the imports to inside the methods. I think that should work

Reply:
Could you check whether `SecretsManager.__init__` is called when you import `get_secret` method?

Reply:
you're right, it does call __init__

Reply:
@wangxiaoyou1993 ok i updated it again. i tested it on a new virtual env and it worked